docs/resources/schema_version.md

```diff
@@ -25,14 +25,11 @@ Schema version resource. None of the fields can be updated. Read more: https://d
 - `schema_id` (String) Schema unique identifier
 
 ## Note
-- Changing any field of the `axual_schema_version` is not allowed, and you should manually delete and create a new `axual_schema_version` with edited values.
 - Please note that Updating the existing `axual_schema_version` is not supported and instead, you should either delete and create an updated `axual_schema_version` or create a new `axual_schema_version` with same schema name, and different version and different schema body.
+- Please note that you might have the permission to delete the schema(as Schema Owner if owner is present) but you might not have the SCHEMA_AUTHOR role(in a Tenant where schema-roles-enforced=true) that is required to create the schema.
 
 ## Limitation
 - `terraform import` functionality is not implemented, will be implemented soon
-- To assign an owner to an existing schema or updating the owner, you need to first delete the `axual_schema_version` and create it again with the updated owner.
-- To update the description, please delete and recreate the schema version.
```

Comment:
Because these lines repeated same thing as this line under "Note":

> Please note that Updating the existing `axual_schema_version` is not supported and instead, you should either delete and create an updated `axual_schema_version` or create a new `axual_schema_version` with same schema name, and different version and different schema body.